libraries/ms-common/src/can.c

```diff
@@ -23,6 +23,13 @@
 
 #define CAN_BUS_OFF_RECOVERY_TIME_MS 500
 
+// flag enabled if x86 passed as platform
+#ifdef X86
+#define TX_CALLBACK_ENABLE 0
+#else
+#define TX_CALLBACK_ENABLE 1
+#endif
```

Comment:
I'd prefer you use `true`/`false` instead of `1`/`0` for booleans (use `#include <stdbool.h>` if you need to). Also explain why you're doing this! Here and/or right above `prv_tx_handler`, put the ticket number (SOFT-301) and briefly explain what the bug is and why this fixes it.